tests/integration/test_smart_contracts.py

```diff
@@ -147,6 +147,11 @@ def test_get_contract_with_contract_id(self):
         self.assertEqual(response.get("status"), 200, response)
         jsonschema.validate(response.get("response"), schema.smart_contract_at_rest_schema)
 
+    def test_get_contract_logs_with_contract_id(self):
+        response = self.client.get_smart_contract_logs(smart_contract_id=SMART_CONTRACT_ARGS_ID)
+        self.assertTrue(response.get("ok"), response)
+        self.assertEqual(response.get("status"), 200)
+
     # LIST #
```

Comment:
You have 3 different parameters for this new endpoint, you should definitely be testing more than just the single required parameter

Reply:
I attempted my best at this. Any suggestion on what to assert against? Not sure the the logs be deterministic

Reply:
For logs, I think schema validation should be good enough